components/BookCard/BookCard.js

```diff
@@ -92,20 +96,30 @@ const StyledInfos = styled.article`
 
 const StyledImage = styled(Image)`
   margin: auto 2px;
+  border-radius: 8px;
+  box-shadow: 0 0 6px 2px rgba(0, 0, 0, 0.15);
 `;
 
 const StyledButton = styled.button`
   background-color: ${({ $isActive }) =>
-    $isActive ? "darkseagreen" : "seashell"};
+    $isActive ? "var(--color-green)" : "var(--color-light-yellow)"};
   border-radius: 4px;
+  border: none;
+  width: 32px;
+  height: 32px;
 `;
 
 const StyledButtonWrapper = styled.section`
   display: flex;
   flex-direction: column;
-  gap: 4px;
+  //gap: 4px;
+  justify-content: space-evenly;
+`;
+
+const StyledTitle = styled.h2`
+  font-size: 20px;
 `;
 
-const StyledGenre = styled.span`
-  font-style: italic;
+const StyledAuthor = styled.p`
+  font-size: 17px;
```

Comment:
Or... `font-size: 1.0625rem`;  😁

```suggestion
  font-size: 1rem;
```